gspread/models.py

```diff
@@ -1500,6 +1509,7 @@ def append_rows(
         params = {
             'valueInputOption': value_input_option,
             'insertDataOption': insert_data_option,
+            'includeValuesInResponse': include_values_in_response,
```

Comment:
@punnerud But in here we still would need custom logic as for example `value_input_option` is not used here. I don't remember the API specs on the top of my head but maybe there are other things like that?

Reply:
Yes, we need to keep every parameter that have logic. All others could be passed directly to the API, and let the API give errors if they don’t exists.

Should make it a bit better than my previous code example, so the parameters can be placed in random order.